lib/plug/cowboy.ex

```diff
@@ -58,6 +58,12 @@ defmodule Plug.Cowboy do
   When using a Unix socket, OTP 21+ is required for `Plug.Static` and
   `Plug.Conn.send_file/3` to behave correctly.
 
+  ## Application configuration
+
+    * `:log_all` - A boolean that determines whether all exceptions are logged by the adapter.
```

Comment:
What if we call it `log_exceptions_with_status_code_from` which defaults to 500? This way people can opt-in to 400 but still leave others out.

Reply:
Sounds good! 👍

Reply:
What if we strechted this and allowed a range or a list? E.g. if I wanted to skip logging 415.

Reply:
`log_exceptions_with_status_code` that expects a list of ranges work for me.